doc/stages/filters.rst

```diff
@@ -206,7 +206,7 @@ the input. These filters will invalidate an existing KD-tree.
 
    filters.crop
    filters.decimation
-   filters.firstinvoxel
+   filters.streamablevoxeldownsize
```

Comment:
I'm not sure it matters a lot, but I do notice an inconsistency in that the files and classes are all StreamableVoxelGrid rather than StreamableVoxelDownsize. I think they should probably match (I prefer the former FWIW).

Reply:
Renamed to StreamableVoxelDownsize everywhere.